olmo/model.py

```diff
@@ -1137,6 +1137,7 @@ def get_alibi_attention_bias(self, seq_len: int, device: torch.device) -> torch.
     def forward(
         self,
         input_ids: torch.LongTensor,
```

Comment:
Ideally I would make `input_ids` optional, but this would not be a fully backwards compatible change.